README.md

```diff
@@ -247,6 +247,18 @@ Some settings can be customized for the entirety of the deployment, they are:
 
 To customize any of these, copy `settings.yaml.example` to `settings.yaml` and add, remove or update the ones you wish to change'
 
+#### Customize Available Boxes
+
+The list of available boxes can be customized by setting an exclude list in `settings.yaml`. This allows faster `vagrant status` calls as well as reducing the the scope of boxes a user sees to tailor to their use cases. To specify boxes to exclude add the following to `settings.yaml`, for example, to remove fips, fedora and any Foreman 1.2X boxes from view:
```

Comment:
Can we be specific and say "`vagrant/settings.yaml` found in forklift's root directory"?